README.md

```diff
@@ -174,3 +174,12 @@ docker-compose build --no-cache matter-server
 docker-compose up -d matter-server
 docker-compose logs -f matter-server
 ```
+
+### Existing docker images
+
+If you wish to use the latest version of this code without maintaining a local image, you can also remove the `build` section in the `docker-compose.yml`, and replace the `image` tag with one of the following, dependent on your architecture:
+ - [aarch64](https://hub.docker.com/r/homeassistant/aarch64-addon-matter-server): `homeassistant/aarch64-addon-matter-server`<sup>
+ - [amd64](https://hub.docker.com/r/homeassistant/amd64-addon-matter-server): `homeassistant/amd64-addon-matter-server`
+
+These are maintained and updated in the [home-assistant/addons](https://github.com/home-assistant/addons/tree/master/matter_server) repository
```

Comment:
@karlshea / @frenck would a note be worthwhile here, or in the addons repo, making the supervisor requirement for those images explicit, so others don't make the same mistake?
I had seen #128, but with some confirming it worked, I must have missed the supervisor requirement later in the comments

Reply:
A Docker compose is provided by this repository:

<https://github.com/home-assistant-libs/python-matter-server/blob/main/docker-compose.yml>

If you want to invent your own method, feel free to do so 🤷

Reply:
Oh, sorry, I wasn't meaning that. I do understand what you're saying, and accept it. 🙂
I meant should the supervisor requirement for the `homeassistant/aarch64-addon-matter-server` and `homeassistant/amd64-addon-matter-server` images be explicitly called out either in this repository, or in the [home-assistant/addons](https://github.com/home-assistant/addons/tree/master/matter_server) repository, so that others don't try to use those images in a non supervisor deployment? I'm trying to help / stop others from making the same mistake without having to look through closed issues / PRs.
It's more a separate suggestion for the README.md docs in one of the repositories.

Reply:
They are add-ons, as the name suggests. Most images under the HA org should not be used (that is not limited to this specific set of images).

Reply:
With the main container image `homeassistant/home-assistant`, and related images also being under the same publisher on docker hub, it can cause confusion that some of the images, as you say, require supervisor, but the main images don't provide it. Because of that, I'm just suggesting a documentation update somewhere to explicitly call this out. 🤷‍♂️.

Reply:
I've not run into these comments or issues before, so I don't think there is a direct need. Please note, the main container image you list in your example isn't our main container image anymore. You can find all the correct information in our documentation.

Or documentation lists what to do, not what not to do.

Reply:
The `homeassistant/home-assistant` on docker hub seems to be the same image (down to the sha256) of that hosted on `ghcr.io`, which is the one recommended via the [docs on the main website](https://www.home-assistant.io/installation/linux#install-home-assistant-container) for container installation, hence the reason I used it simply as an example. This was in reference to you mentioning

> Most images under the HA org should not be used

People will look under the `homeassistant` publisher for other images they may use.

The suggestion I was making, was simply to whether it was worth while noting what situation the images _should_ be used / are intended for. This would both help people know what setup they need to use those images, and help avoid issues / queries such as this, and those discussed in #128, that may arise. I certainly wouldn't suggest maintaining a list all the things you shouldn't do, as that list may become excessive, and I agree, doesn't seem worth while.

Reply:
> People will look under the homeassistant publisher for other images they may use.

As said above, we haven't had that issues before.

> The suggestion I was making, was simply to whether it was worth while noting what situation the images should be used / are intended for.

Yes got that, but as said, we should document what one should do. Not what one shouldn't do.

Reply:
> As said above, we haven't had that issues before.

What I'm raising is it **has** been an issue before, of people finding images under the `homeassistant` publisher and trying to use them, expressly in #128 (see [one of the first comments](https://github.com/home-assistant-libs/python-matter-server/issues/128#issuecomment-1342110358) is a prime example), and has required [previous clarification](https://github.com/home-assistant-libs/python-matter-server/issues/128#issuecomment-1349138522). Marcel mentioned in his response he had copied from another issue of someone also making the same mistake. Obviously Karl referenced this in his message response above.

> The issue had been closed by our maintainers with the message the request isn't becoming available right now because of a lack of bandwidth.

I do understand that issue #128 has been closed, and why it was closed, and fully accept the reasoning behind this.

> Yes got that, but as said, we should document what one should do. Not what one shouldn't do.

As mentioned before, I'm not suggesting adding anything about what one shouldn't do. Simply the requirements for running one of these add-on containers, similar to how the [requirements for running Home Assistant Supervised](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) are mentioned in it's [installation instructions](https://github.com/home-assistant/supervised-installer/blob/main/README.md).

I just thought a couple of lines somewhere mentioning the HA Supervisor requirement would be useful to people, and avoid further issues / PRs being raised. I'll leave this as is going forward.

Please do take my appreciation for Home Assistant as a whole, it's a really useful project.

Reply:
The installation methods and their general differences are already documented here:

https://www.home-assistant.io/installation/#compare-installation-methods

I think it's pretty clear that add-ons require a Supervisor, either via OS or Supervised installation if you read that comparison, and it's explicitly mentioned if you follow the link for the add-ons where there's a note. Why would you think you can run the add-ons on another installation method when we say that's not possible in the docs?